### PR TITLE
mm: Fix /proc/<pid>/smaps output format

### PR DESCRIPTION
**Pull request by FooIbar**
It breaks dumpsys meminfo.

Fixes: bf71e8c

### CODE REVIEW

File: fs/proc/task_mmu.c [hunk 1]
```diff
@@ -1182,7 +1181,6 @@ static int show_smaps_rollup(struct seq_file *m, void *v)
 
 	show_vma_header_prefix(m, priv->mm->mmap ? priv->mm->mmap->vm_start : 0,
 			       last_vma_end, 0, 0, 0, 0);
-	seq_pad(m, ' ');
```
Comment (adithya2306) [inline fs/proc/task_mmu.c:1185]: why is this necessary? removing line 1067 seems to be enough to fix it

Comment (FooIbar) [reply fs/proc/task_mmu.c:1185]: It's not necessary for the fix, but the extra space is also redundant since space is already added here.
https://github.com/ghostrider-reborn/android_kernel_common/blob/26f1f5d8e19d8e9f87ab952fed2668275889a480/fs/proc/task_mmu.c#L457

**Comment by adithya2306** [comment]
https://github.com/ghostrider-reborn/android_kernel_common/commit/97bddd8476a0ca9b4a47ea8eca549700169fc126
Thanks
